adyenv6core/src/com/adyen/v6/exceptions/AdyenNonAuthorizedPaymentResultException.java

```diff
@@ -24,24 +24,24 @@
 import com.adyen.model.checkout.PaymentsResponse;
 import com.adyen.model.terminal.TerminalAPIResponse;
 
-public class AdyenNonAuthorizedPaymentException extends Exception {
```

Comment:
What is the reason to rename it?

Reply:
As discussed, this is an additional class to support paymentdetailsresponse type. AdyenNonAuthorizedPaymentResultException is already handling more than one types. Can you please check if we can use existing class?

Reply:
Added the field paymentsDetailsResponse in AdyenNonAuthorizedPaymentException